README.md

```diff
@@ -1,4 +1,4 @@
-<img src="https://storage.googleapis.com/passage-docs/passage-logo-gradient.svg" alt="Passage logo" style="width:250px;"/>
+<img src="https://storage.googleapis.com/passage-docs/Lockup%20Vertical%20color.png" alt="Passage logo" style="width:250px;"/>
```

Comment:
IDK if this is just me being nitpicky but is it weird to be encoding spaces in here vs. having a snake case file name?

Reply:
I think someone from design/product team made that file :)